src/float.cr

```diff
@@ -223,6 +228,11 @@ struct Float64
     value.to_f64
   end
 
+  # Returns an `Float64` by invoking `unsafe_to_f64` on *value*.
+  def Float64.unsafe_new(value)
```

Comment:
```suggestion
  def self.unsafe_new(value)
```

Reply:
The change was done that way to minimize the diff. In some cases the type is mention explicitly and in others `self` is used.